.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,15 @@
+name: Pull Request
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: PR - cp*, ubuntu-22.04
+    runs-on: ubuntu-22.04
```

Comment:
I'd do `ubuntu-latest`, AFAIK they only use stable versions, i.e. next one will be 24.04